src/models/comment.js

```diff
@@ -7,13 +7,13 @@ const {
 
 const commentSchema = new Schema(
   {
-    text: {
-      type: String
-    },
     author: {
       type: Schema.Types.ObjectId,
       ref: USER
     },
+    text: {
+      type: String
+    },
```

Comment:
Why did these fields change their order?

Reply:
Thanks for your comment! The reason for changing the order in the schema was to prioritize certain fields, such as `author` and `text`, in the response data:
<img width="495" alt="Screenshot 2024-04-30 at 15 45 41" src="https://github.com/Space2Study-UA-1156/BackEnd-02/assets/28622400/b3d722e7-0401-4310-be5f-874362445272">
I want to receive the exact expected result, where the `text` field in the comments model should come after the `author` field, but I find that `text` is positioned before `author`, for this situation I should create a new model. However, if I notice that we are using this model for both getting and adding comments, and we specifically require the `text` field to be positioned after `author`, I handle the situation accordingly.

Reply:
I think the order of the fields in the response is not important, but we can change it. Thank you!